src/raftstore/store/fsm/apply.rs

```diff
@@ -2448,8 +2449,22 @@ impl ApplyFsm {
                 self.delegate.pending_cmds.append_normal(cmd);
             }
         }
-        // TODO: observe it in batch.
-        APPLY_PROPOSAL.observe(propose_num as f64);
+
+        APPLY_PROPOSAL_LOCAL.with(|hist| {
```

Comment:
@BusyJay 

maybe it is better to let these thread pool support OnTick mechanism?

Reply:
@siddontang A OnTick mechanism implemented,please see the newest code